<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1581</width>
    <height>1177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    box-shadow: 0px 0px 20px rgba(0, 0, 0, 0.5);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>218</red>
         <green>218</green>
         <blue>218</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>218</red>
         <green>218</green>
         <blue>218</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="96">
         <red>164</red>
         <green>166</green>
         <blue>168</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <widget class="QTableWidget" name="tableWidgetListaDeContatos">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>600</y>
      <width>1501</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <column>
     <property name="text">
      <string>CPF</string>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>NOME COMPLETO</string>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>EMAIL</string>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TELEFONE</string>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ENDEREÇO</string>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_Telefone">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>430</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:700; color:#000000;&quot;&gt;TELEFONE :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_CEP">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>340</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; color:#000000;&quot;&gt;CEP&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_CEP">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>340</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
   border: 2px solid black;
    border-radius: 10px;
}
QLineEdit:focus {
    color: red;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_CPF">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; color:#000000;&quot;&gt;CPF&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCPF">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
   border: 2px solid black;
    border-radius: 10px;
}
QLineEdit:focus {
    color: red;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNomeCompleto">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>1051</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
   border: 2px solid black;
    border-radius: 10px;
}
QLineEdit:focus {
    color: red;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_NomeCompleto">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; color:#000000;&quot;&gt;NOME COMPLETO&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditEmail">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>260</y>
      <width>651</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
   border: 2px solid black;
    border-radius: 10px;
}
QLineEdit:focus {
    color: red;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Email">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>260</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; color:#000000;&quot;&gt;EMAIL&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Logradouro">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; color:#000000;&quot;&gt;RUA&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditLogradouro">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>661</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
   border: 2px solid black;
    border-radius: 10px;
}
QLineEdit:focus {
    color: red;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Numero">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>340</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; color:#000000;&quot;&gt;NÚMERO&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_NumeroCasa">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>340</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
   border: 2px solid black;
    border-radius: 10px;
}
QLineEdit:focus {
    color: red;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_DDI">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>480</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; color:#000000;&quot;&gt;DDI&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDDI">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>480</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
   border: 2px solid black;
    border-radius: 10px;
}
QLineEdit:focus {
    color: red;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_DDD">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>480</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; color:#000000;&quot;&gt;DDD&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDDD">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>480</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
   border: 2px solid black;
    border-radius: 10px;
}
QLineEdit:focus {
    color: red;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNumero">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>480</y>
      <width>661</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
   border: 2px solid black;
    border-radius: 10px;
}
QLineEdit:focus {
    color: red;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_NumeroTelefone">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>480</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; color:#000000;&quot;&gt;NÚMERO&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAlterar">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>540</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 2px solid black;
   background-color: red;
    border-radius: 10px;
}

QPushButton:pressed {
    background-color: white;
    color: black;
}

QPushButton:released {
    background-color: black;
    color: white;
}
</string>
    </property>
    <property name="text">
     <string>ALTERAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Fundo Icones Programas/icons8-macho-de-configurações-de-admin-30.png</normaloff>../../../Fundo Icones Programas/icons8-macho-de-configurações-de-admin-30.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConsultar">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>540</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 2px solid black;
   background-color: red;
    border-radius: 10px;
}

QPushButton:pressed {
    background-color: white;
    color: black;
}

QPushButton:released {
    background-color: black;
    color: white;
}
</string>
    </property>
    <property name="text">
     <string>CONSULTAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Fundo Icones Programas/icons8-procurar-usuário-masculino-26.png</normaloff>../../../Fundo Icones Programas/icons8-procurar-usuário-masculino-26.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonIncluir">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>540</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 2px solid black;
   background-color: red;
    border-radius: 10px;
}

QPushButton:pressed {
    background-color: white;
    color: black;
}

QPushButton:released {
    background-color: black;
    color: white;
}
</string>
    </property>
    <property name="text">
     <string>INCLUIR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Fundo Icones Programas/icons8-usuário-50.png</normaloff>../../../Fundo Icones Programas/icons8-usuário-50.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>496</number>
    </property>
    <property name="autoRepeatInterval">
     <number>500</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonExcluir">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>540</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     border: 2px solid black;
   background-color: red;
    border-radius: 10px;
}

QPushButton:pressed {
    background-color: white;
    color: black;
}

QPushButton:released {
    background-color: black;
    color: white;
}
</string>
    </property>
    <property name="text">
     <string>EXCLUIR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Fundo Icones Programas/icons8-descarte-de-lixo-50.png</normaloff>../../../Fundo Icones Programas/icons8-descarte-de-lixo-50.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLimpar">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>540</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 2px solid black;
   background-color: red;
    border-radius: 10px;
}

QPushButton:pressed {
    background-color: white;
    color: black;
}

QPushButton:released {
    background-color: black;
    color: white;
}
</string>
    </property>
    <property name="text">
     <string>LIMPAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Fundo Icones Programas/icons8-zelador-50.png</normaloff>../../../Fundo Icones Programas/icons8-zelador-50.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:700; font-style:italic;&quot;&gt;Agenda Contatos 2023&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1480</x>
      <y>0</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 2px solid black;
   background-color: red;
    border-radius: 0px;
}

QPushButton:pressed {
    background-color: white;
    color: black;
}

QPushButton:released {
    background-color: black;
    color: white;
}
</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>0</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 2px solid black;
   background-color: red;
    border-radius: 0px;
}

QPushButton:pressed {
    background-color: white;
    color: black;
}

QPushButton:released {
    background-color: black;
    color: white;
}
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-8</x>
      <y>-1</y>
      <width>1631</width>
      <height>1281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../Fundo Icones Programas/Fundo.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>tableWidgetListaDeContatos</zorder>
   <zorder>label_Telefone</zorder>
   <zorder>label_CEP</zorder>
   <zorder>lineEdit_CEP</zorder>
   <zorder>label_CPF</zorder>
   <zorder>lineEditCPF</zorder>
   <zorder>lineEditNomeCompleto</zorder>
   <zorder>label_NomeCompleto</zorder>
   <zorder>lineEditEmail</zorder>
   <zorder>label_Email</zorder>
   <zorder>label_Logradouro</zorder>
   <zorder>lineEditLogradouro</zorder>
   <zorder>label_Numero</zorder>
   <zorder>lineEdit_NumeroCasa</zorder>
   <zorder>label_DDI</zorder>
   <zorder>lineEditDDI</zorder>
   <zorder>label_DDD</zorder>
   <zorder>lineEditDDD</zorder>
   <zorder>lineEditNumero</zorder>
   <zorder>label_NumeroTelefone</zorder>
   <zorder>pushButtonAlterar</zorder>
   <zorder>pushButtonConsultar</zorder>
   <zorder>pushButtonIncluir</zorder>
   <zorder>pushButtonExcluir</zorder>
   <zorder>pushButtonLimpar</zorder>
   <zorder>label_2</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1581</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
